install.sh

```diff
@@ -30,6 +30,18 @@ else
     echo "Skip installing ROS"
 fi
 
+if [ "$ros2_distro" '==' "foxy" ]; then
```

Comment:
You also need to check whether the architecture is x64 or not.

Reply:
Now there is a check process before OpenVINO installation to ensure that CPU vendor is Intel.